care/facility/static_data/icd11.py

```diff
@@ -1,43 +1,28 @@
 import contextlib
 import json
 
+from django.db import connection
 from littletable import Table
 
+from care.facility.models.icd11_diagnosis import ICD11Diagnosis
+
 
 def fetch_data():
     with open("data/icd11.json", "r") as json_file:
         return json.load(json_file)
```

Comment:
this is only being used in the management command so it would be better to move it there